### ISSUE TITLE
Change to LocaleListener breaks Symfony 3.4 LTS

### ISSUE THREAD

**Issue by jayesbe** [Prezent/doctrine-translatable-bundle#28]
https://github.com/Prezent/doctrine-translatable-bundle/pull/27

This PR broke our 3.4 LTS app with the error

```
FatalThrowableError

Type error: Argument 1 passed to Prezent\Doctrine\TranslatableBundle\EventListener\LocaleListener::onKernelRequest() must be an instance of Symfony\Component\HttpKernel\RequestEvent, instance of Symfony\Component\HttpKernel\Event\GetResponseEvent given, called in ... Symfony/Component/EventDispatcher/Debug/WrappedListener.php on line 115
```

**Comment by sandermarechal** [comment]
Should be fixed in 1.2.1, apologies for the inconvenience.

**Comment by jayesbe** [comment]
Thanks @sandermarechal
